templates/terraform/custom_import/index_self_link_as_name_set_project.go.erb

```diff
@@ -0,0 +1,19 @@
+
+	config := meta.(*Config)
+
+	// current import_formats can't import id's with forward slashes in them.
```

Comment:
I'm not sure what this means- don't we import ids with slashes all the time?

Reply:
Fixed this + where I copied that bit from. We can't handle a field with `/`'s in the value, such as `name`s like `accessPolicies/{{short_name}}`